provisioner/puppet-masterless/provisioner.go

```diff
@@ -26,6 +26,9 @@ type Config struct {
 	// Additional arguments to pass when executing Puppet
 	ExtraArguments []string `mapstructure:"extra_arguments"`
 
+	// The Guest OS Type (unix or windows)
+	GuestOSType string `mapstructure:"guest_os_type"`
```

Comment:
Can we make this ordering alphabetical?

Reply:
Happy to have it alphabetical. But the primary motivation was to normalize the section across puppet-masterless and puppet-server.

Reply:
FWIW pretty much none of the other provisioners sort their Config structs. :)

Reply:
yeah, but adding a little code hygiene when you make changes never hurt anyone. I'm not going to block your PR on this, it's just nice to do

Reply:
datastructures alphabetized